feature/experimental/system/gnmi/benchmarking/ate_tests/drained_configuration_convergence_time/drained_configuration_convergence_time_bgp_test.go

```diff
@@ -73,6 +74,10 @@ func setMED(t *testing.T, dut *ondatra.DUTDevice, d *oc.Root) {
 	actions5 := stmt.GetOrCreateActions()
 	setMedBGP := actions5.GetOrCreateBgpActions()
 	setMedBGP.SetMed = oc.UnionUint32(bgpMED)
+	actions5.PolicyResult = oc.RoutingPolicy_PolicyResultType_ACCEPT_ROUTE
```

Comment:
Please add condition here to accept only bgp routes.  Otherwise it will accept all routes in RIB and causes issue.

stc := stmt.GetOrCreateConditions()
stc.InstallProtocolEq = oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP

We have added similar way in https://github.com/openconfig/featureprofiles/blob/main/feature/experimental/bgp/ate_tests/bgp_remove_private_as/bgp_remove_private_as_test.go

Reply:
Same change is needed in setASPath function. Thanks

Reply:
Using this is currently a source of contention between different vendors since there are two ways to do the same thing(https://github.com/openconfig/public/issues/931).

Anyways, I am now filtering only on the BGP routes advertised by Port 1.

Reply:
@Ankur19  gotMED is filtered based on prefixes but we still import all routes into BGP.  "isConverged" is called but exits after max iterations are completed.  We never hit pass criteria.  But scripts works as gotMED is having entries equal to routecount. Is it possible to add one more check to see if gotMED equal to routeCount  ?